### ISSUE TITLE
Version incompatibilities

### ISSUE THREAD

**Issue by tvogels** [disneyresearch/jeri#13]
This project requires a fixed version of react and react-dom (of all dependencies, in fact). This leads to warnings when using JERI in a project that features a more recent version of the framework.

```
[3/4] Linking dependencies...
warning " > jeri@1.1.0" has incorrect peer dependency "react@15.6.1".
warning " > jeri@1.1.0" has incorrect peer dependency "react-dom@15.6.1".
```

**Comment by cbreak-black** [comment]
What would be a better solution? I was told that webdevs always fix their dependencies (and even commit thousands of lines worth of npm-shrinkwrap.json and yarn.lock files) because of the fragility of the ecosystem. Can we selectively loosen the requirements or should we just delete the lock files?

**Comment by ja-no** [comment]
Why not simply turning the peer dependencies into dependencies? AFAIK this would bundle the necessary versions with JERI and would not require the developer to handle the peer dependencies. If we do so, we should also edit the examples and remove the explicit loading of react and react-dom from cdnjs.cloudflare.com.

**Comment by cbreak-black** [comment]
This should be 'fixed' in e5e0c021e62e9f3ed44dce72428258aa141dcb88 and 21ed42ab297e077ecccd1940a590965c0958a755, to be released eventually. The fix is to insist on a newer version.
There seem to be technical reasons to make react and react-dom peer instead of direct dependencies: [see this react issue on the topic](https://github.com/facebook/react/issues/2402) and [this related issue](https://github.com/ReactTraining/react-router/issues/6724) with [further explanations in the yarn docs](https://yarnpkg.github.io/berry/advanced/peer-dependencies/).
